code/modules/clothing/suits/heavy_armor.dm

```diff
@@ -111,7 +111,8 @@
 	icon_state = "tribal_power_armor"
 	item_state = "tribal_power_armor"
 	body_parts_covered = CHEST|GROIN|ARMS|LEGS
-	armor = list("melee" = 65, "bullet" = 55, "laser" = 55, "energy" = 10, "bomb" = 30, "bio" = 40, "rad" = 20, "fire" = 40, "acid" = 0, "wound" = 25)
+	armor = list("melee" = 70, "bullet" = 70, "laser" = 60, "energy" = 10, "bomb" = 30, "bio" = 40, "rad" = 20, "fire" = 40, "acid" = 0, "wound" = 25)
+	slowdown = 1.75
```

Comment:
Slowdown is useless because it's the same as it's parent one.

Reply:
fixed